docs/en/deploy-loom-testnet.md

```diff
@@ -1,22 +1,22 @@
 ---
 id: deploy-loom-testnet
-title: Deploy to Loom Testnet
-sidebar_label: Deploy to Loom Testnet
+title: Deploy to Extdev Testnet
+sidebar_label: Deploy to Extdev Testnet
 ---
 
 ## Who Should Read This Document
 
-This doc is for developers, and it aims to show how they can join the Loom Testnet and deploy their smart contracts. If you are looking for information on how to run nodes or to become a validator, please checkout our [Validator Guide](validator.html).
+This doc is for developers, and it aims to show how they can join the Extdev Testnet and deploy their smart contracts. If you are looking for information on how to run nodes or to become a validator, please checkout our [Validator Guide](validator.html).
 
 ## Overview
 
-Deploying to Loom PlasmaChain is no different from deploying to the Ethereum main net, or to Rinkeby. If you know how to do one, you also know how to do the other.
+Deploying to Basechain is no different from deploying to Ethereum mainnet, or to Rinkeby. If you know how to do one, you also know how to do the other.
 
-We at Loom are using Truffle to build, test, and deploy our smart contracts. To make our life easier, we developed something called a provider that lets Truffle deploy to Loom PlasmaChain just like it deploys to Ethereum.
+Loom Network uses Truffle to build, test, and deploy our smart contracts. To make our life easier, we developed something called a provider that lets Truffle deploy to Basechain just like it deploys to Ethereum.
 
-Without delving too much into details, the provider works like a bridge that makes Web3.js calls compatible with Loom PlasmaChain.
+Without delving too much into details, the provider works like a bridge that makes Web3.js calls compatible with Basechain.
 
-This document shows how you can use `Truffle` and the [Loom Truffle Provider](https://github.com/loomnetwork/loom-truffle-provider) to build a simple Web UI that interacts with Loom PlasmaChain.
+This document shows how you can use `Truffle` and the [Loom Truffle Provider](https://github.com/loomnetwork/loom-truffle-provider) to build a simple Web UI that interacts with Basechain.
```

Comment:
This page is supposed to be about deploying to Extdev, why does it go on and on about Basechain?

Reply:
Updated. However, I have a question. I'll post in slack.